### ISSUE TITLE
🆕 Software Suggestion | Tresorit (file storage)

### ISSUE THREAD

**Issue by udf2457** [privacytools/privacytools.io#2032]
## Basic Information

Tresorit
File Storage
https://tresorit.com/

## Description

Fantastic secure zero-knowledge file storage service based in Switzerland.

They also have a free, secure file sending service https://send.tresorit.com/

**Comment by dngray** [comment]
This would likely be relevant to https://github.com/privacytools/privacytools.io/pull/1713

**Comment by udf2457** [comment]
As a side note, I did see that the prior attempt by someone else at listing Tresorit was closed in favour of Mozilla. Which struck me as odd because for starters Mozilla is very much US-based and no doubt uses US servers whilst Tresorit is 110% Swiss and uses servers in far more favourable jurisdiction.

**Comment by subsys-R9boq8** [comment]
>I did see that the prior attempt by someone else at listing Tresorit was closed in favour of Mozilla.

Probably because Firefox Send is FOSS/FLOSS and is self-host friendly?

**Comment by udf2457** [comment]
> Probably because Firefox Send is FOSS/FLOSS and is self-host friendly?

Still doesn't detract from the hard fact that they are a US organisation and their servers are largely US based. 

You can't have it both ways.  You can't rant on about "five eyes" and mass surveillance (https://www.privacytools.io/classic/) and yet only promote service options provided by US organisations.

**Comment by subsys-R9boq8** [comment]
>Still doesn't detract from the hard fact that they are a US organisation and their servers are largely US based.

Yeah if you're not satisfied with that you can just _git clone_ and make your own instance in some country not included in the 5/9/14 eyes, that's my main reason to recommend FF send instead of Tresorit (though probably not very normal-user-friendly).

**Comment by blacklight447-ptio** [comment]
Sseeing how Firefox Send is now dead, this seems like a reasonable service to include. Even if its closed source. Firefox send was browser encryption anyway, so it was always hard to trust the encryption if you didn't self host it. @privacytools/editorial

**Comment by dotjersh** [comment]
Tresorit is a software product. Wouldn't it make sense to include it under [Cloud Storage](https://www.privacytools.io/providers/cloud-storage/)? It's a very valuable alternative to iCloud, Dropbox, and GDrive. Since it's based in Switzerland and is end to end encrypted, I feel like it matches the criteria of privacytools.io very well.

**Comment by lrq3000** [comment]
Can Tresorit be self-hosted? If not, that's likely the reason why it was not included on PTIO. If you notice, most sections only include self-hostable services, except of course for services where it's not possible or very impractical, eg, hosting.

**Comment by joshgrift** [comment]
It is not self hosted, but I do think that privacytools.io has lots of services that aren't self hosted though, such as their [email](https://privacytools.io/providers/email/) provider list.

**Comment by lrq3000** [comment]
Yes but as I said that's the exception. It depends on the open-source offering. Currently Nextcloud is listed in cloud providers, which has lots of hosters for those who don't want to self-host, but are powered by an open-source and potentially self-hostable software for those who want maximum privacy. For file sharing, there are much more flexible open-source solutions with unlimited file size (OnionShare, MuWire, Magic Wormhole, ShareDrop).

Note I'm not part of the PTIO team, the above reflect only my own opinion.
